juju/juju
# Rename the branch command to add-branch.

## Description of change

Rename branch command to add-branch.

## QA steps

1. export JUJU_DEV_FEATURE_FLAGS=generations
2. juju bootstrap
3. juju branch                     <- ERROR unrecognized command
3. juju add-branch new     <- success
3. check 'see also' of other branch related commands
4. juju help add-branch

## Documentation changes

This feature is still under development, however any docs referencing juju branch should be updated to use juju add-branch in the develop stream.
